scripts/update_library_version.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+# Check if the script is running on Linux
+if [[ "$OSTYPE" != "linux-gnu"* ]]; then
+  echo "Error: This script can only be run on Linux."
```

Comment:
One thing we can consider in the future is to run this script in a container to make it independent of the host environment.